### PR TITLE
Configure git

### PR DESCRIPTION
**Pull request by dzhlobo**
See more:
- http://haacked.com/archive/2014/07/28/github-flow-aliases/
- https://git-scm.com/docs/git-config

### CODE REVIEW

**Comment by dzhlobo** [comment]
@roman-dubrovsky, what do you think about it?

File: gitconfig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[color]
+  ui = true
+[fetch]
+  prune = true
+[pull]
+  rebase = true
+[alias]
+  hist = log --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --graph --date=short
+  lg = !git --no-pager log --graph --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --abbrev-commit --date=short -20
+  lga = !git --no-pager log --graph --pretty=tformat:'%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%ar)%Creset [%an]' --abbrev-commit --date=short -20 --all
```
Comment (roman-dubrovsky) [inline gitconfig]: @Proghat  мб лучше lga через lg запишим?

File: gitconfig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  cob = checkout --track -b
+  st = status --short
+  pr = pull --rebase
+  pom = push origin master
+  up = !git pull --rebase --prune $@ && git submodule update --init --recursive
+  wipe = !git add -A && git commit -qm 'WIPE SAVEPOINT' && git reset HEAD~1 --hard
+  bclean = "!f() { git branch --merged ${1-master} | grep -v " ${1-master}$" | xargs git branch -d; }; f"
+  bdone = "!f() { git checkout ${1-master} && git up && git bclean ${1-master}; }; f"
+[include]
+  path = ~/.gitconfig.local
```
Comment (roman-dubrovsky) [inline gitconfig:27]: @Proghat  про гит много чего здесь обсуждалось https://github.com/datarockets/dotfiles/issues/3

File: gitconfig (hunk 1 above)
Comment (dzhlobo) [inline gitconfig]: Right. Good idea!

**Comment by evgenTraytyak** [comment]
Git aliases: https://github.com/datarockets/dotfiles/issues/3

File: gitconfig (hunk 2 above)
Comment (dzhlobo) [inline gitconfig:27]: I don't like to duplicate all aliases here. My idea is to put very common (`ci`, `co`) and complicated aliases in `.gitconfig` such as `lg`, `wipe`. And simple common aliases to `.aliases`. What do you think about separation?

**Comment by dzhlobo** [comment]
@evgenTraytyak, right. I saw #3. But I'd like to put complicated aliases in `.gitconfig` and then use it in `.aliases`:

```
alias gl="git lg"
```

**Comment by evgenTraytyak** [comment]
@Proghat :+1:

**Comment by roman-dubrovsky** [comment]
oh my god, I forgot that we speak in English on this repo :cry:

File: gitconfig (hunk 2 above)
Comment (roman-dubrovsky) [inline gitconfig:27]: @Proghat  I dont understand what do you mean

**Comment by evgenTraytyak** [comment]
@Proghat good :shipit:

File: gitconfig (hunk 2 above)
Comment (dzhlobo) [inline gitconfig:27]: @roman-dubrovsky, should I duplicate all git aliases here? I don't think so. What aliases you would like to add to that gitconfig also?

Comment (roman-dubrovsky) [inline gitconfig:27]: @Proghat  i think we can add all git aliases to gitconfig

**Pull request merged**